bin/deploy.sh

```diff
@@ -1,7 +1,7 @@
-git branch | grep "* master" > /dev/null
+git branch | grep "* main" > /dev/null
```

Comment:
this script is unused and I have a PR to delete it. https://github.com/covid-projections/covid-projections/pull/2453